### PR TITLE
added /api endpoint with suitable tests to successfully respond with a…

### PR DESCRIPTION
**Pull request by TomaszSarmata**
… full list of available endpoints

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -38,3 +38,30 @@ describe("GET: /api/topics", () => {
...
+        endpointsArr.forEach((endpoint) => {
+          expect(endpoint).toMatchObject({
+            description: expect.any(String),
+            queries: expect.any(Array),
+            exampleResponse: expect.any(Object),
+          });
+        });
+      });
+  });
+  test("404: ERROR - responds with the path is not found", () => {
```
Comment (NC-Edd-C) [inline __tests__/api.test.js:59]: This is a good test but deals with any invalid endpoint so shouldn't be in the Get /api description box

File: controllers/api.controllers.js [hunk 2]
```diff
@@ -10,4 +10,11 @@ const getAllTopics = (req, res, next) => {
     });
 };
 
-module.exports = { getAllTopics };
+const getAllEndpoints = (req, res, next) => {
+  //   console.log("hello from controller");
```
Comment (NC-Edd-C) [inline controllers/api.controllers.js:14]: Console.logs

File: models/api.models.js [hunk 3]
```diff
@@ -12,4 +13,12 @@ const fetchAllTopics = () => {
     });
 };
 
-module.exports = fetchAllTopics;
+const fetchAllEndpoints = () => {
+  return fs.readFile("./endpoints.json").then((results) => {
+    const parsedResults = JSON.parse(results);
+    // console.log(parsedResults, "results here");
```
Comment (NC-Edd-C) [inline models/api.models.js:19]: console.log

**Review by NC-Edd-C — COMMENTED** [review]
Very good, some comments but nothing major

**Comment by TomaszSarmata** [comment]
thanks very much Edd, I've just removed the unnecessary test for 404

**Pull request merged**